src/pages/Login/Login.js

```diff
@@ -1,7 +1,182 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Modal from 'react-modal';
 
 function Login() {
-  return <div>Login</div>;
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const ModalHandler = () => {
+    setModalOpen(prev => !prev);
+  };
+  return (
+    <>
+      <button onClick={ModalHandler} />
+
+      <Modal
+        isOpen={modalOpen}
+        onRequestClose={ModalHandler}
+        style={modalStyle}
+      >
+        <ModalTop>
+          <Logo src="images/login/needed.png" alt="logo" />
```

Comment:
```suggestion
          <Logo src="/images/login/needed.png" alt="logo" />
```
절대경로로 적어주세요! 상대경로로 지정하게 되면 path의 depth가 깊어졌을때 원하는대로 public 폴더에 접근을 하지 못합니다!

예시) 상대경로로 작성했을 시
- http://localhost:3000/main => 이경우에는 상대경로로 작성해도 public 폴더로 정상 인식해서 정상동작
- http://localhost:3000/detail/1 => 이경우에는 path의 depth가 깊어져서 상대경로로 지정하게되면 pubilc/detail 폴더안에서 찾으려고 하기 때문에 제대로 동작하지 않음.